### ISSUE TITLE
A request for dataset of paper

### ISSUE THREAD

**Issue by Yuqi99** [mobarakol/Learning_WhereToLook#1]
Dear prof. Islam,
       l am a graduate student from Changchun university of science and technology in China.
       My research is focused on machine learning and saliency prediction. I have recently read your paper"Learning Where to Look While Tracking Instruments in Robot-assisted Surgery",which appeared in"Miccai 2019".I am very interested in this article and have studied it carefully. I would like to attempt to predict saliency maps in surgical scenarios, but the training sets 1, 2, 3, 5, and 6, 8 ,9, 10 are not publicly available. The dataset for the 2017 MICCAI Surgical Instrument Segmentation Challenge only contains ground truth segmentation masks and does not include ground truth saliency maps. I was wondering if you could kindly send me the complete dataset and necessary information. I assure you that this information will only be used for research purposes.
Thank you very much for your kind consideration and I am looking forward to your early reply.
Sincerely,
      From Yuqi Yan
